docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,885 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #60.        for _ in range(n_chunks_upperbound):    
</pre><p>Originally, there was a while loop. I replace it with a for-loop and add a couple of <code>break</code> statements to terminate the loop. Note that the number of iterations shoud be, at least, equal to the number of chunks in <code>T[:query_idx]</code>  So, one easy answer  is <code>query_idx</code> since it is definitely larger than the number of chunks. But, just to add some sense to the code and make it a bit more readable, I computed a tighter upperbound for total number of chunks. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>It is now changed to make the code closer to the proposed algorithm</p>